src/core/lib/event_engine/ares_resolver.cc

```diff
@@ -524,15 +522,17 @@ void AresResolver::OnHostbynameDoneLocked(void* arg, int status,
     std::string error_msg =
         absl::StrFormat("address lookup failed for %s: %s",
                         hostname_qa->query_name, ares_strerror(status));
-    GRPC_ARES_RESOLVER_TRACE_LOG("request:%p on_hostbyname_done_locked: %s",
-                                 ares_resolver, error_msg.c_str());
+    GRPC_ARES_RESOLVER_TRACE_LOG("OnHostbynameDoneLocked: %s",
```

Comment:
I thought since now we use 1 AresResolver for all types of requests (for one client channel that is, maybe there would be multiple AresResolvers for multiple client channels? e.g. client wants to send RPCs to different services?), there would be no need to log the value of its pointer. Though I have added it back in case it would be useful in some case.

Reply:
Yes, I was thinking about the case where there are multiple channels in the same binary.  That case is quite common, and without seeing which resolver each log message is associated with, the logs can be almost impossible to understand.